tests/expectations/core/test_expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,194 @@
+from typing import Final
+
+import pandas as pd
+import pytest
+from contrib.experimental.great_expectations_experimental.expectations.expect_column_values_to_be_present_in_other_table import (
+    ExpectColumnValuesToBePresentInOtherTable,  # needed for expectation registration
+)
+
+import great_expectations.exceptions as gx_exceptions
+from great_expectations.compatibility.sqlalchemy_compatibility_wrappers import (
+    add_dataframe_to_db,
+)
+from great_expectations.core import ExpectationConfiguration
+from great_expectations.data_context.util import file_relative_path
+from great_expectations.datasource.fluent import SqliteDatasource
+
+DB_PATH: Final[str] = file_relative_path(
+    __file__, "../../test_sets/referential_integrity_dataset.db"
+)
```

Comment:
let's maybe `resolve` and also assert that it exists?

Reply:
`resolve` is already handled by the `file_relative_path()` function

https://github.com/great-expectations/great_expectations/blob/9dfecf4d27781b53fd89d8589bcad34becd061c1/great_expectations/data_context/util.py#L134